src/main/java/org/dependencytrack/tasks/vulnerabilitypolicy/blobstorage/GenericStorageHandler.java

```diff
@@ -0,0 +1,6 @@
+package org.dependencytrack.tasks.vulnerabilitypolicy.blobstorage;
+
+import org.dependencytrack.tasks.vulnerabilitypolicy.AbstractVulnerabilityPolicyFetcher;
+
+public class GenericStorageHandler extends AbstractVulnerabilityPolicyFetcher {
```

Comment:
Do we need this class? It looks like `downloadZippedContent` always returns `null`, but `verifyDownloadNeeded` always returns `true`. Code using this handler will most definitely break.